From 78fda9afeea1e99e034be3c02494096713c17f0c Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Sat, 19 Dec 2020 10:12:40 -0500
Subject: [PATCH 2/2] HAXFIX: no time for that!

  Prebuilt apk found in PRODUCT_COPY_FILES: $(1), use BUILD_PREBUILT
  instead blablabla

Change-Id: Ia30e92df22e343f9dd61850aefaa573fbcb3b825
---
 core/Makefile | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/core/Makefile b/core/Makefile
index 571ce50c8..45753a1df 100644
--- a/core/Makefile
+++ b/core/Makefile
@@ -14,9 +14,6 @@ LOCAL_PATH := $(BUILD_SYSTEM)
 #$(1): the src:dest pair
 #$(2): the dest
 define check-product-copy-files
-$(if $(filter-out $(TARGET_COPY_OUT_SYSTEM_OTHER)/%,$(2)), \
-  $(if $(filter %.apk, $(2)),$(error \
-     Prebuilt apk found in PRODUCT_COPY_FILES: $(1), use BUILD_PREBUILT instead!))) \
 $(if $(filter true,$(BUILD_BROKEN_VINTF_PRODUCT_COPY_FILES)),, \
   $(if $(filter $(TARGET_COPY_OUT_SYSTEM)/etc/vintf/% \
                 $(TARGET_COPY_OUT_SYSTEM)/manifest.xml \
-- 
2.32.0 (Apple Git-132)

